crates/hdk/src/lib.rs

```diff
@@ -165,6 +165,10 @@
 //!   - Close runs when an agent is deprecating an old source chain in favour of a new one.
 //!   - All zomes in a DNA migrate at the same time.
 //!   - Any failure fails the migration.
+//! - `fn recv_remote_signal(signal: SerializedBytes) -> ExternResult<()>`:
```

Comment:
```suggestion
//! - `fn recv_remote_signal(signal: ExternIO) -> ExternResult<()>`:
```

Reply:
I double-checked that the [payload](https://github.com/holochain/holochain/blob/3e67466bc5d6f633382fa872cbe491ab040e8089/crates/holochain_p2p/src/spawn/actor.rs#L1380) sent [over the wire](https://github.com/holochain/holochain/blob/3e67466bc5d6f633382fa872cbe491ab040e8089/crates/holochain_p2p/src/spawn/actor.rs#L1402) is `ExternIO`:

Reply:
ah okay, so there's some documentation elsewhere that's wrong -- the docs for `send_remote_signal` shows an example using `SerializedBytes`. That'll require a follow-up commit.

Reply:
Great, thanks for checking :+1: